Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -283,6 +283,9 @@ public enum CommandLineArgumentsResult {
           }
           dafnyFiles.Add(df);
           isDafnyFile = true;
+        } catch (ArgumentException e) {
+          options.Printer.ErrorWriteLine(options.ErrorWriter, "*** Error: " + e.Message);
+          return CommandLineArgumentsResult.PREPROCESSING_ERROR;
```

Comment:
Instead of using exceptions for control flow here, I'd actually prefer we do the same thing as in the preceding if branch: check `if (!File.Exists(filePath))` first and print out the same error message, before attempting the `DooFile.Read` (which then doesn't have to change).

Reply:
I put in the check for File.Exists, but left the exception catching.
- things can happen in the file system between checking that a file exists and reading it
- in reading/unzipping the .doo file all sorts of bad things can happen, many of which are not worth trying to predict